CONTRIBUTING.md

```diff
@@ -1,86 +1,187 @@
-# Raising issues
+# Contributing to Pact Python
 
-_Before raising an issue, please ensure that you are using the latest version of pact-python._
+Pact Python is the Python implementation of the [Pact](https://pact.io) integration testing framework. If you're interested in contributing to Pact Python, hopefully, this document makes the process for contributing clear.
 
-Please provide the following information with your issue to enable us to respond as quickly as possible.
+The [Open Source Guides](https://opensource.guide/) website has a collection of resources for individuals, communities, and companies who want to learn how to run and contribute to an open source project. Contributors and people new to open source alike will find the following guides especially useful:
 
-- The relevant versions of the packages you are using.
-- The steps to recreate your issue.
-- The full stacktrace if there is an exception.
-- An executable code example where possible. You can fork this repository and
-  use the [examples] directory to quickly recreate your issue.
+-   [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
+-   [Building Welcoming Communities](https://opensource.guide/building-community/)
```

Comment:
There is also a devrel repo as well

https://github.com/pact-foundation/devrel

which was previously named `readme` which can serve as a central hub 

Sorry for the link spam! Not necessarily asking you to action anything off this, but just to consider, provide a map of some of things and get your thoughts on how we can corale this to make a more cohesive experience not just here in pact-python but provide a similar experience in places people expect (being a newbie helps bring fresh perspective :) )

Reply:
I found the `opensource.guide` links to be quite good and general introduction to contributing to open source, but definitely I need to add the `docs.pact.io/contributing` links! I'm not sure yet how to incorporate the `devrel` repo, as it isn't as clear what the user should do once they get there.

Reply:
>I found the opensource.guide links to be quite good and general introduction to contributing to open source, but definitely I need to add the docs.pact.io/contributing links! 

I will give them a read! I am sure they will be useful with others, and its something that resonated with you it will with others.


>I'm not sure yet how to incorporate the devrel repo, as it isn't as clear what the user should do once they get there.

Yeah it's not really a home, or a springboard to anything really, just a bit of copy pasta from the docs, possibly for users who aren't using the docs website.

Open to suggestions and ways we can improve it! or if it is even useful!